src/guides/v2.4/graphql/mutations/move-items-between-requisition-list.md

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
Same comment here. The mutation is now named `moveItemsBetweenRequisitionList`, and there have been numerous schema changes.